internal/bminventory/inventory.go

```diff
@@ -2037,6 +2037,10 @@ func (b *bareMetalInventory) updateNetworkParams(params installer.UpdateClusterP
 		setMachineNetworkCIDRForUpdate(updates, machineCidr)
 	}
 	if params.ClusterUpdateParams.NetworkType != nil && params.ClusterUpdateParams.NetworkType != cluster.NetworkType {
+		// If user
```

Comment:
please remove this remark or complete the sentence

Reply:
No need for comment it is self explanatory